static/scripts/rewards/render-transaction/read-claim-data-from-url.ts

```diff
@@ -60,12 +61,10 @@ export async function readClaimDataFromUrl(app: AppState) {
      */
   }
 
-  displayRewardDetails();
-  displayRewardPagination();
+  await displayRewardsWithDetails();
 
-  await renderTransaction();
-  if (app.networkId !== null) {
-    await verifyCurrentNetwork(app.networkId);
+  if (app.claims[0].networkId !== null) {
+    await verifyCurrentNetwork(app.claims[0].networkId);
```

Comment:
With stacked permits this check would need to be added into the claim flow so that we confirm the network for each potentially different claim